package.json

```diff
@@ -33,5 +33,10 @@
   "engines": {
     "node": ">=0.10"
   },
-  "private": true
+  "private": true,
+  "dependencies": {
+    "backbone": "^1.1.2",
+    "jquery": "^2.1.4",
```

Comment:
One issue we had with this PR is that this dependency will result in a second version of JQuery being included if the JQuery 1.x branch is being used (which we're using for IE8 support). It then breaks IE8 functionality because this version of JQuery is used in these instances.

Could this be changed to something more like

```
"backbone": "^1.1.2",
"jquery": "*", // or "jquery": ">= 1.9.0" (as a guess)
"underscore": "^1.8.3"
```